test_plans/sycl_ext_oneapi_composite_device.asciidoc

```diff
@@ -0,0 +1,128 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for sycl_ext_oneapi_composite_device
+
+THis is a test plan for the API described in the 
```

Comment:
```suggestion
This is a test plan for the API described in the 
```

Reply:
Applied in f01911d3f691d211d876b02fd6afb7c876d33851